### PR TITLE
List of themes and view theme pages

### PR DESCRIPTION
**Pull request by apradillap**
Connects to #829

### CODE REVIEW

File: config/locales/gobierto_participation/views/es.yml [hunk 1]
```diff
@@ -22,6 +22,29 @@ es:
           donde debatiremos en persona sobre los temas de este proceso. Si no puedes
           asistir, iremos publicando online todo lo que se hable en la reuniones para
           facilitar el seguimiento y hacer accesibles las conclusiones.
+    issues:
+      current_group:
+        current_stage: Fase actual
+      index:
+        intro: Tus ideas ayudan a mejorar %{location_name} y la vida de todos los
```
Comment (ferblape) [inline config/locales/gobierto_participation/views/es.yml]: This should be `intro_html` to don't escape the HTML:

screen shot 2017-08-18 at 09 00 50

File: app/controllers/gobierto_participation/issues_controller.rb [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    def index
+      @issues = current_site.issues.alphabetically_sorted
+    end
+
+    def show
+      @issue = find_issue
+      @issue_news = find_issue_news
+      @issue_news_updated = find_issue_news_updated
+      @issue_events = find_issue_events
+      @processes = current_site.processes.process.open
```
Comment (ferblape) [inline app/controllers/gobierto_participation/issues_controller.rb]: Is there any reason you are not filtering the processes and the groups that belong to the current issue?

File: app/controllers/gobierto_participation/issues_controller.rb [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+      @groups = current_site.processes.group_process
+    end
+
+    private
+
+    def find_issue
+      Issue.find_by_slug!(params[:id])
```
Comment (ferblape) [inline app/controllers/gobierto_participation/issues_controller.rb]: We should always filter by site. Two sites can have issues with the same slug.

File: test/integration/gobierto_participation/issues/issue_show_test.rb [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+    end
+
+    def test_issue_without_events
+      with_current_site(site) do
+        visit @path
+
+        assert has_content? "There are no related events"
+      end
+    end
+  end
```
Comment (ferblape) [inline test/integration/gobierto_participation/issues/issue_show_test.rb]: I miss a test that checks the processes and the groups of this issue.

File: app/views/gobierto_participation/processes/show.html.erb [hunk 5]
```diff
@@ -74,12 +74,10 @@
...
           <div class="pure-g meta_info">
             <div class="pure-u-1 pure-u-md-1-4">
-              <span class="accent"><i class="fa fa-question-circle"></i><%= t('.issues') %></span>
+              <span class="accent"><i class="fa fa-question-circle"></i><%= t('.issue') %></span>
             </div>
             <div class="pure-u-1 pure-u-md-1-2">
-              <%= todo_link_to do %>
-                <% link_to @process.issue.name, '' %>
-              <% end %>
+              <%= link_to @process.issue.name, gobierto_participation_issue_path(@process.issue) %>
```
Comment (ferblape) [inline app/views/gobierto_participation/processes/show.html.erb]: The link should use the `slug` not the `id` of the issue.

**Review by ferblape — CHANGES_REQUESTED** [review]

File: config/locales/gobierto_participation/views/es.yml (hunk 1 above)
Comment (apradillap) [reply config/locales/gobierto_participation/views/es.yml]: Fix it!! https://github.com/PopulateTools/gobierto/pull/868/commits/f9b7849cd40f84a39111766616d065ded802053e

File: app/controllers/gobierto_participation/issues_controller.rb (hunk 2 above)
Comment (apradillap) [reply app/controllers/gobierto_participation/issues_controller.rb]: No, I forgot it

https://github.com/PopulateTools/gobierto/pull/868/commits/f1c3ee240e5bb4930543e445d9033a03be6d8126

File: app/controllers/gobierto_participation/issues_controller.rb (hunk 3 above)
Comment (apradillap) [reply app/controllers/gobierto_participation/issues_controller.rb]: https://github.com/PopulateTools/gobierto/pull/868/commits/4086b134aa46c779f25ed6d0342caa933e3073a4

I must have present multitenant, I reviewed other files

File: app/views/gobierto_participation/processes/show.html.erb (hunk 5 above)
Comment (apradillap) [reply app/views/gobierto_participation/processes/show.html.erb]: Fix 
https://github.com/PopulateTools/gobierto/pull/868/commits/bc292e52644e987f570f9a79f610446ef31485b2

File: test/integration/gobierto_participation/issues/issue_show_test.rb (hunk 4 above)
Comment (apradillap) [reply test/integration/gobierto_participation/issues/issue_show_test.rb]: https://github.com/PopulateTools/gobierto/pull/868/commits/806a4d8136f845a0ed7049f9a2e0bd81f38bc0da

**Review by ferblape — APPROVED** [review]

**Pull request merged**